generator/internal/descriptor_utils.cc

```diff
@@ -430,6 +430,10 @@ void SetHttpGetQueryParameters(
 
 std::string DefaultIdempotencyFromHttpOperation(
     google::protobuf::MethodDescriptor const& method) {
+  if (method.name() == "GetIamPolicy" ||
```

Comment:
nit: there is prior code to detect `SetIamPolicy` methods, maybe you should have followed the same structure as those?